### ISSUE TITLE
Error loading map in QGIS

### ISSUE THREAD

**Issue by zannisara** [maptiler/qgis-maptiler-plugin#197]
Hello, I'm trying to import the Outdoor map in QGIS 3.38.1 but most of the style was not generated. These are the details of the error:

> The following warnings were generated while converting the vector tile style:
> transport_bus:` Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_cave: Could not convert expressions in layer style definition, check Python console for details.
> mountain_peak: Could not convert expressions in layer style definition, check Python console for details.
> water_name_point: Could not convert expressions in layer style definition, check Python console for details.
> road_label_ferry: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_tourist_info: Could not convert expressions in layer style definition, check Python console for details.
> place_state: Could not convert expressions in layer style definition, check Python console for details.
> water_name_line: Could not convert expressions in layer style definition, check Python console for details.
> transport_airfield: Could not convert expressions in layer style definition, check Python console for details.
> place_capital: Could not convert expressions in layer style definition, check Python console for details.
> country_other: Could not convert expressions in layer style definition, check Python console for details.
> contour_label: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_camp_site: Could not convert expressions in layer style definition, check Python console for details.
> transport_airport: Could not convert expressions in layer style definition, check Python console for details.
> place_city: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_church: Could not convert expressions in layer style definition, check Python console for details.
> place_park: Could not convert expressions in layer style definition, check Python console for details.
> water_name_way: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_memorial_monument: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_hut: Could not convert expressions in layer style definition, check Python console for details.
> place_other: Could not convert expressions in layer style definition, check Python console for details.
> transport_railway: Could not convert expressions in layer style definition, check Python console for details.
> continent: Could not convert expressions in layer style definition, check Python console for details.
> country_rank_3: Could not convert expressions in layer style definition, check Python console for details.
> viaferrata_symbol: Could not convert expressions in layer style definition, check Python console for details.
> road_label: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_castle: Could not convert expressions in layer style definition, check Python console for details.
> country_rank_1-2: Could not convert expressions in layer style definition, check Python console for details.
> place_village: Could not convert expressions in layer style definition, check Python console for details.
> mountain_volcano-us: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_fortress: Could not convert expressions in layer style definition, check Python console for details.
> mountain_volcano: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_viewpoint: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_shelter: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_ruins: Could not convert expressions in layer style definition, check Python console for details.
> place_town: Could not convert expressions in layer style definition, check Python console for details.
> road_label_track: Could not convert expressions in layer style definition, check Python console for details.
> outdoor_poi_waterfall: Could not convert expressions in layer style definition, check Python console for details.
> place_region: Could not convert expressions in layer style definition, check Python console for details.
> mountain_peak-us: Could not convert expressions in layer style definition, check Python console for details.

I've checked the Python console but it doesn't say anything. Any ideas about how I can fix this issue?
Thanks a lot for your help!

Sara

**Comment by lazaa32** [comment]
Duplicate of #193 
Fixed in #195 

Hi Sara, there were some changes made in QGIS 3.38 that have affected label rendering in maps. It's not related just to Outdoor map but almost to every map. The problem was identified and fixed in the latest version `v3.3.1`. This version was released yesterday and is waiting for approval.

The solution for your problem is simple. Wait for the latest version of the plugin to be approved (usually within few days). Once the version is approved in plugin repository, you will be notified in QGIS and you will be able to upgrade. The latest version of the plugin should display Outdoor map completely without any warnings.

**Comment by zannisara** [comment]
Cool! Thank you very much: this was very helpful. BTW: great plugin! I love it!
